### DISCUSSION TITLE
不知道你们下载的时候会不会出现

### DISCUSSION THREAD

**Discussion by wuwenjun9939** [Meloong-Git/PCL#3562] [讨论]
![image](https://github.com/Hex-Dragon/PCL2/assets/110022109/f6004f18-4d47-4a99-9c68-1692e26698bd)
下载源码（公开的2.6.15和2.6.14版）的时候微软提示不安全，想问一下，什么情况？

**Comment by burningtnt** [comment]
误报不是一天两天了……

**Comment by wuwenjun9939** [comment]
@LTCatt 龙腾还没解决吗？

**Comment by wuwenjun9939** [comment]
而且，不光是微软报毒，就连国内杀软的火绒，也报
![image](https://github.com/Hex-Dragon/PCL2/assets/110022109/3827aa7f-0bd6-4510-81c4-27cb6c29002c)

**Comment by SqrtTan** [comment]
源码加壳了，容易误报

**Comment by wuwenjun9939** [comment]
怪不得

**Comment by Silverteal** [comment]
可能由于语义上存在文件系统和注册表相关操作，火绒将源码识别为了危险VBS脚本，实际上并不是VBS，只是VB而已。
